test/CadlRanchProjectsNonAzure/type/property/optionality/src/Generated/Models/PlainTimeProperty.Serialization.cs

```diff
@@ -0,0 +1,135 @@
+// <auto-generated/>
```

Comment:
Same as above

Reply:
To confirm - the new test cases for these new specs from cadl ranch are not included in this PR, right?